src/error/mod.rs

```diff
@@ -104,6 +108,11 @@ impl RoverError {
         {
             return check_response.get_json();
         }
+        if let Some(RoverClientError::LintFailures { lint_response }) =
+            self.error.downcast_ref::<RoverClientError>()
+        {
+            return lint_response.get_json();
+        }
```

Comment:
same here, this should be a `match` statement now

Reply:
Moved to using a `match` statement